input/docs/png_vs_jpg.md

```diff
@@ -0,0 +1,90 @@
+title: PNG vs JPG
+description: Comparing .png and .jpg formats
+---
+>This document works with MATLAB built-in image *coloredChips.png* in `A` variable. The code uses `A` to create images in an **imgs/** folder.
+# Creating an image database
+* to compare atributes of both formats, following code will create database of 100 **.jpg** and 100 **.png** images of *coloredChips.png*, varying in size. 
+* First create **imgs/** folder in your actual working directory
+``` matlab
+clear all 
+
+route= 'imgs/'; %sets route to saving directory
+A=imread('coloredChips.png'); %reads an image
+
+for i=1:100
+	pic_res=imresize(A,i/100); %img resize
+	[r, c]=size(pic_res);
+	pic_size(i,1)=r*c; %size of current img in pixels
+
+	imwrite(pic_res,[route ,'pic_' num2str(i/100) '.jpg'],'JPEG'); %saving current img in .png
+	imwrite(pic_res,[route, 'pic_' num2str(i/100) '.png'],'PNG'); %saving current img in .jpg
+end
+
+```
+#Loading of .jpg images
```

Comment:
DRY (don't repeat yourself). Vím, takto je to v původním dokumentu a přesně si pamatuji, jak jsem si říkal že až to budu někdy přepisovat, musím to nějak upravit.. Teď to padlo na tebe. Zkus to udělat nějak tak, ať se kód neopakuje pro každý formát.  Zřejmě si udělat buňky s formáty `formats = {'.jpg' , '.png' }; ` a ten kód obalit fórem, který bude iterovat po formátech...